syntax = "proto3";

message SizeProtoModel {
    int32 size = 1;
    float price = 2;
    int32 stock = 3;
}

message ProductItemFormProtoModel {
    int32 product_id = 1;
    string color = 2;
    string image_url = 3;
    repeated SizeProtoModel sizes = 4;
}
